plugins/services/src/js/containers/service-connection/SDKServiceConnectionEndpointList.js

```diff
@@ -17,24 +18,23 @@ import EndpointClipboardTrigger from "./EndpointClipboardTrigger";
 import ServiceNoEndpointsPanel from "./ServiceNoEndpointsPanel";
 import SDKEndpointActions from "../../events/SDKEndpointActions";
 import SDKEndpointStore from "../../stores/SDKEndpointStore";
-import { EDIT } from "../../constants/ServiceActionItem";
-import ServiceActionDisabledModal
-  from "../../components/modals/ServiceActionDisabledModal";
 
 class SDKServiceConnectionEndpointList extends React.Component {
   constructor() {
     super(...arguments);
 
     this.state = {
-      actionDisabledModalOpen: false,
       servicePreviousState: ""
     };
   }
 
-  handleOpenEditConfigurationModal(actionDisabledModalOpen) {
-    this.setState({
-      actionDisabledModalOpen
-    });
+  handleOpenEditConfigurationModal() {
```

Comment:
Let's change the binding in L203 so it doesn't pass in `true` unnecessarily. Let's keep a clean house :)